docs/python_docs/python/tutorials/performance/backend/mkldnn/mkldnn_readme.md

```diff
@@ -62,18 +62,18 @@ To achieve better performance, the Intel OpenMP and llvm OpenMP are recommended
 ```
```

Comment:
Old MKL-DNN names starting from line 18.
Maybe the directory and file name could be also changed

Reply:
Changed the names, but left the file name. I believe it should be changed later if the file nomenclature will change from mkldnn to onednn in whole project.